Doc/Makefile

```diff
@@ -1,24 +1,28 @@
 subdirs :=
-
+BP_VERSION := 1.78
```

Comment:
Can you do something like ``python -c "import Bio; print(Bio.__version__)"`` to get the current version installed?

Or better, something like ``python -c "`grep ^__version__  ../Bio/__init__.py`; print(__version__)"`` or ``grep ^__version__  ../Bio/__init__.py` | cut -f 2 -d '"'`` to get the version in the current tree?

Reply:
I had considered something like this but wanted to be modest with my changes since I already added an extra file (`version.sh`) to the build system to get Hevea to work. I also thought about adding the `\today` macro to `Tutorial.tex` to insert the date when the Tutorial was built. This would mean that building the tutorial for releases would become entirely automatic  - should I add this back in or is that being _too_ bold?